nilearn/regions/parcellations.py

```diff
@@ -92,16 +110,19 @@ def _labels_masker_extraction(img, masker, confound):
 class Parcellations(MultiPCA):
     """Learn parcellations on fMRI images.
 
-    Four different types of clustering methods can be used such as kmeans,
-    ward, complete, average. Kmeans will call MiniBatchKMeans whereas
-    ward, complete, average are used within in Agglomerative Clustering.
-    All methods are leveraged from scikit-learn.
+    Five different types of clustering methods can be used:
+    kmeans, ward, complete, average and rena.
+    kmeans will call MiniBatchKMeans whereas
+    ward, complete, average are used within in Agglomerative Clustering and
+    rena will call ReNA.
+    kmeans, ward, complete, average are leveraged from scikit-learn.
+    rena is buit into nilearn.
 
     .. versionadded:: 0.4.1
 
     Parameters
     ----------
-    method : str, {'kmeans', 'ward', 'complete', 'average'}
+    method : str, {'kmeans', 'ward', 'complete', 'average', 'rena'}
```

Comment:
```suggestion
    method: str, {'kmeans', 'ward', 'complete', 'average', 'rena'}
```

Reply:
Also, here we should give some advice on how to choose which method. Here is the paragraph that I suggest:

```
For a small number of parcels, kmeans is usually advisable. For a large number of parcellations (several hundreds, or thousands), ward and rena are the best options. Ward will give higher quality parcels, but with increased computation time. Rena is most useful as a fast data-reduction step, typically dividing the signal size by ten.
```